### ISSUE TITLE
missing images

### ISSUE THREAD

**Issue by rimmesbe** [AASOxRevolution/prj-rev-bwfs-dasmoto#5]
Looks like you are missing some images here. I would also recommend using the alt attribute on your image tags for accessibility: https://www.codecademy.com/articles/accessibility

https://github.com/AASOxRevolution/prj-rev-bwfs-dasmoto/blob/master/Dasmotos%20Arts%20and%20Crafts/index.html#L13-L18
